config/samples/dataplane_v1beta1_openstackdataplane_baremetal.yaml

```diff
@@ -53,7 +53,8 @@ spec:
         ansibleUser: cloud-admin
         ansiblePort: 22
         ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
-        ansibleVars: |
+        ansibleVars:
+          nova: {}
```

Comment:
isn't it supposed to be under `node`/`nodeTemplate`?
https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/api/v1beta1/common.go#L83

Reply:
yeah, thanks for spotting!